API.md

```diff
@@ -1004,7 +1020,7 @@ item: {
 }
 ```
 
-For instance, if you have specified the data like this 
+For instance, if you have specified the data like this
```

Comment:
I'm surprised about these line ending changes. This can cause the doc checking step to randomly fail.